src/admin/mod.rs

```diff
@@ -52,9 +56,22 @@ pub struct Server {
     server: hyper::server::Builder<hyper::server::conn::AddrIncoming>,
     workload_info: WorkloadInformation,
     registry: Arc<Mutex<Registry>>,
+    config: Config,
     shutdown_trigger: signal::ShutdownTrigger,
 }
 
+#[derive(serde::Serialize, Debug, Clone)]
+pub struct ConfigDump {
+    #[serde(flatten)]
+    workload_info: WorkloadInformation,
+    #[serde(default)]
+    static_workloads: Vec<Workload>,
+    #[serde(default)]
+    version: String,
```

Comment:
nit: would be kind of nice to make this a struct so we can get a json representation

Reply:
done